adobe/spectrum-web-components
# Low Hanging Fruit

## Description
- use latest VRT URL in README.md
- remove trailing comma in Number Field Docs
- Allow Number Field (and Editable Slider) to display values while `step=0`
- Prevent crash when submitting a large number to a Number Field (and Editable Slider)
- Correct usage of `<sp-divider vertical>` in documentation
- Ensure single upload dialog in Dropzone documentation
- Update linking to Spectrum CSS

## Related issue(s)
- fixes #2014 
- fixes #2013
- fixes #1998
- fixes #1995
- fixes #1981

## Motivation and context
Lots of small things fixed!

## How has this been tested?

    1. Go [here](https://fixes--spectrum-web-components.netlify.app/components/number-field/#minimum-and-maximum-values)
    2. Type `123456789012335457` into the "Red value" Number Field
    3. See the page NOT crash
    1. Go [here](https://fixes--spectrum-web-components.netlify.app/components/number-field/#minimum-and-maximum-values)
    2. In DevTools set the `step` for the "Red value" Number Field to `0`
    3. Set the `value` for the "Red value" Number Field to `5`
    4. See that `5` is displayed.
    1. Go [here](https://fixes--spectrum-web-components.netlify.app/components/dropzone/#example)
    2. Click "Select a File"
    3. See that the Upload Dialog is only opened once.
    4. Click "from your computer"
    5. See that the Upload Dialog is only opened once.
    1. Go [here](https://fixes--spectrum-web-components.netlify.app/components/divider/#vertical)
    2. See that all three sizes of `<sp-divider vertical>` fill the height of their sibling Icons
    1. Go [here](https://fixes--spectrum-web-components.netlify.app/components/number-field/)
    2. See that there are no errors in the console.

## Types of changes

## Checklist
